src/utils/Constants.js

```diff
@@ -0,0 +1 @@
+export const PULL_REQUEST_ICON_URL = "https://visualpharm.com/assets/817/Pull%20Request-595b40b85ba036ed117da8b8.svg";
```

Comment:
Are you using this, if so, download the svg and store it into assets, otherwise the load times for extension will depend on network.

Reply:
removed